src/missiondef.cpp

```diff
@@ -311,6 +313,7 @@ mission_type_id mission_type::from_legacy( int old_id )
     static const std::vector<mission_type_id> old_id_vec = {{
         mission_type_id( "MISSION_NULL" ),
         mission_type_id( "MISSION_GET_ANTIBIOTICS" ),
+        mission_type_id( "MISSION_GET_INHALER" ),
```

Comment:
This vector is for legacy purposes only, it must not be changed or old missions will not load.